src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -1392,6 +1392,9 @@ private ICollection<P> AddBuiltInProperties()
 
 #if RUNTIME_TYPE_NETCORE
             builtInProperties.Add(SetBuiltInProperty(ReservedPropertyNames.msbuildRuntimeType, "Core"));
+#elif MONO
+            builtInProperties.Add(SetBuiltInProperty(ReservedPropertyNames.msbuildRuntimeType,
+                                                        NativeMethodsShared.IsMono ? "Mono" : "Full"));
```

Comment:
Under what circumstance is the `MONO` symbol defined, but `NativeMethodsShared.IsMono` returns false?